plop-templates/Component/stories.tsx.hbs

```diff
@@ -0,0 +1,19 @@
+import { storiesOf } from "@storybook/react";
+import React from "react";
+import { IntlProvider } from "react-intl";
+
+import { MemoryRouter } from "react-router";
+import { {{pascalCase name}} } from ".";
+import { I{{pascalCase name}}Props } from "./{{pascalCase name}}";
+
+const DEFAULT_PROPS: I{{ pascalCase name }}Props = {};
+
+storiesOf("@components/{{ componentType }}/{{pascalCase name}}", module)
+.addParameters({ component: {{pascalCase name}} })
+.addDecorator(story => (
+<IntlProvider locale="en">
+  <MemoryRouter initialEntries={["/"]}>{story()}</MemoryRouter>
```

Comment:
Does it really belong here?
Components should get navigation callbacks from parents, if we need router in mocked environment it's a signal we're doing something wrong.